src/ChunkReader.php

```diff
@@ -93,6 +95,13 @@ public function read(WithChunkReading $import, Reader $reader, TemporaryFile $te
 
         $jobs->push($afterImportJob);
 
+        // Check if the import class is batchable
+        if ($import instanceof ShouldBatch) {
+            return Bus::batch([
+                $jobs->toArray(),
```

Comment:
This would create a batch with a chain, is that your intention? Would it make sense to add a concern to allow in parallel?

Reply:
Yes, that's right

Reply:
the concern is added so that the response is a PendingBatch and then you can add things like batch name, jobs at the end of the batch etc.